back-end/handlers/hostHelpers.ts

```diff
@@ -96,7 +88,7 @@ const hostStartQuizTimer = async (gameId: number, io: Server): Promise<void> =>
   nextQuestionTimer = setTimeout(hostPreAnswer, timePerQuestionMS, gameId, io);
 }
 
-const hostShowNextQuestion = async (gameId: number, io: Server): Promise<void> => {
+const hostNextQuestionOrLeaderboard = async (gameId: number, io: Server): Promise<void> => {
```

Comment:
rename this to best reflect what it does - goes to a question OR the final leaderboard